src/richie/plugins/simple_text_ckeditor/defaults.py

```diff
@@ -1,5 +1,17 @@
 """Default settings for Richie's simple text plugin."""
 
 from django.conf import settings
+from django.utils.translation import gettext_lazy as _
 
 SIMPLETEXT_CONFIGURATION = getattr(settings, "RICHIE_SIMPLETEXT_CONFIGURATION", [])
+
+SIMPLETEXT_VARIANTS = getattr(
+    settings,
+    "RICHIE_SIMPLETEXT_VARIANTS",
+    [
+        ("", _("None")),
+        ("undefined", _("Undefined variant")),
+        ("blue-on-white", _("Blue on white")),
+        ("white-on-blue", _("White on blue")),
+    ],
+)
```

Comment:
This comment is related to my overall comment.

Make variant more generic
```suggestion
SIMPLETEXT_VARIANTS = getattr(
    settings,
    "RICHIE_SIMPLETEXT_VARIANTS",
    [
        ("", _("None")),
        ("undefined", _("Box")),
        ("fulfill", _("Fulfilled box")),
        ("stroked", _("Stroked box")),
    ],
)
```